src/Gulpfile.js

```diff
@@ -13,10 +13,28 @@ var glob = require("glob"),
 	uglify = require("gulp-uglify"),
 	rename = require("gulp-rename"),
     concat = require("gulp-concat"),
-	header = require("gulp-header")
+	header = require("gulp-header"),
+    notify = require("gulp-notify")
 
 /*
 ** GULP TASKS
+---------------------------------------
+   Checks Themes, Modules, and Core directories for an Assets.json file. 
+   Assets.json defines what Less, CSS, TypeScript, JS files should be processed by Gulp.
+   
+   When defining your own Assets.json file, it should be saved in the root of module or theme project
+
+   Assets.json example:
+   Saved to /Modules/My.Custom.Module/Assets.json
+   [
+      {
+        "inputs": [ "Less/master.less" ], //Specifies which files to process during the Build task
+        "output": "Styles/@.css", //When @ is specified, each file specified in "inputs" will be converted to [filename].css
+        "watch": ["Less/*.less"], //specifies which files to watch, use this when you have a master Less file that imports other Less files
+        "generateSourceMaps": true //Will include source maps in unminified version, defaults to 'true'
+      }
+    ]
+    "inputs" and "output" are required.  All other properties are optional.
```

Comment:
This comment should go. We're going to write documentation for this, and also include a commented sample `Assets.json` in module scaffolding code-gen.